### PR TITLE
DashboardScene: Adds solo page that uses dasboarde scene to render single panel

### PR DESCRIPTION
**Pull request by torkelo**
* [x] SoloPanelPage that loads the dashboard but renders just single panel using all space available 
* [x] Adds feature toggle dashboardsSceneSolo 
* [x] handle timezone query param 
* [x] Handle panel repeats (Very tricky)
* [x] Handle row repeats (Very tricky)
* [x] new E2E tests

Scenes does not support old urls for repeated panels, this is so far one of the big "breaking changes" 

* Old url like `panelId=15`, needs to become `panelId=panel-2-clone-2`   (with not real way to migrate/translate/redirect them) 
* for repeated panels inside a repeated row it's `panelId=panel-2-row-2-clone-2`

TODO in follow up PRs
* [ ] Some checks that the image renderer does will not work https://github.com/grafana/grafana-image-renderer/blob/master/src/browser/browser.ts#L341 
* [ ] Need to increment window.panelsRendered somewhere as [that is what image renderer checks](https://github.com/grafana/grafana-image-renderer/blob/master/src/browser/browser.ts#L355C36-L355C50)
* [ ] Enterprise reporting and [repeated panels support ](https://github.com/grafana/grafana-enterprise/pull/1241) needs a major rethink or removal

### CODE REVIEW

**Comment by Eve832** [comment]
Just peeked at this PR, can you please decide whether to capitalize **Scenes** ; seeing it written both ways :)

File: public/app/routes/routes.tsx [hunk 1]
```diff
@@ -85,8 +85,10 @@ export function getAppRoutes(): RouteDescriptor[] {
       pageClass: 'dashboard-solo',
       routeName: DashboardRoutes.Normal,
       chromeless: true,
-      component: SafeDynamicImport(
-        () => import(/* webpackChunkName: "SoloPanelPage" */ '../features/dashboard/containers/SoloPanelPage')
+      component: SafeDynamicImport(() =>
+        config.featureToggles.dashboardScene || config.featureToggles.dashboardSceneSolo
```
Comment (harisrozajac) [inline public/app/routes/routes.tsx:89]: Just curious, what would be the case when only `dashboardSceneSolo` is enabled? It looks like as long as `dashboardScene` is enabled solo will work too

**Review by harisrozajac — APPROVED** [review]
Looks great! Just left one question out of curiosity

File: public/app/routes/routes.tsx (hunk 1 above)
Comment (torkelo) [reply public/app/routes/routes.tsx:89]: I guess this should be config.featureToggles.dashboardScene && config.featureToggles.dashboardSceneSolo

**Pull request merged**